### ISSUE TITLE
Abnormal Large Task Timings

### ISSUE THREAD

**Issue by philltomlinson** [ansible/ansible#13129]
On the v2.0.0-0.3.beta1 tag I'm currently getting abnormally large task timings (all appear slightly different but start with 1447240...):

```
2015-11-11 11:32:28,585 p=6905 u=vagrant |  TASK: Task 1 ------------------------1447240566.82s
2015-11-11 11:32:28,586 p=6905 u=vagrant |  TASK: Task 2 ----------------------- 1447240365.34s
2015-11-11 11:32:28,586 p=6905 u=vagrant |  TASK: Task 3 ----------------------- 1447240335.68s
2015-11-11 11:32:28,586 p=6905 u=vagrant |  TASK: Task 4 ----------------------- 95.47s
```

However some of the tasks do appear to log an accurate time. I'm running in CentOS 6.7. I've been unable to identify a pattern in terms of if it is related to a specific module or action for a given task. It almost appears to be random. Wondering if anyone else has seen this on the Dev branch?

**Comment by jimi-c** [comment]
@philltomlinson could you try the beta2 or the current devel branch? I think we may have addressed this there, if not let us know.

**Comment by philltomlinson** [comment]
Just tried this on the Devel branch and I'm still getting very large timings, even on tasks that skip because they fail a when condition.

**Comment by nitzmahone** [comment]
Suspect this could be related to #13271 - @philltomlinson is this a custom callback plugin that's giving those numbers or something internal? Those large numbers look like raw time.time() output to me (which could be cause by multiply calling start/stop callback logic that was meant to only be called once)...

**Comment by bcoca** [comment]
@philltomlinson what are you using to get those timings?

**Comment by philltomlinson** [comment]
This can now be closed as mentioned above its a duplicate of #13271
